Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/entrycard.vue

```diff
@@ -22,6 +27,17 @@ export default class EntrycardTimelineComponent extends Vue {
     @Prop({ default: true }) allowComment!: boolean;
     @Prop({ default: true }) showDetailsButton!: boolean;
 
+    @Getter("getEntryComments", { namespace: "comment" })
+    entryComments!: (entyId: string) => UserComment[];
+
+    private get comments(): UserComment[] {
```

Comment:
Note: This method is never used outside of being called by another the getter 'commentsCount.

Reply:
merged 2 methods